config-overrides.js

```diff
@@ -1,12 +1,12 @@
 // const ModuleScopePlugin = require("react-dev-utils/ModuleScopePlugin");
-import ModuleScopePlugin from "react-dev-utils/ModuleScopePlugin";
+const ModuleScopePlugin = require("react-dev-utils/ModuleScopePlugin");
 
-export default function override(config, _environment) {
+module.exports = function override(config, _environment) {
```

Comment:
Codacy has a fix for the issue: [Do not use "module".](https://app.codacy.com/gh/devntv/useHooks_Apply_projects/pullRequest?prid=9726020)
```suggestion
export default function override(config, _environment) {
```